blog/2024-01-12-pulsar-2023-year-in-review.md

```diff
@@ -0,0 +1,120 @@
+
+Since Pulsar’s graduation as a Top-Level Project (TLP) in September 2018, it has been driven by an active global community, with **12K+** commits from **639** contributors, **12.2K+** stars, **3.5K** forks, and **10K+** Slack users.
+
+We are grateful to all of our community members and those in the broader open-source community who contributed to the Apache Pulasr project. They are the reasons behind every step Apache Pulsar has made over the past years.
+
+Now, let’s take a look at some of the highlights in 2023.
+
+# 2023 Highlights
+
+## Apache Pulsar 3.0 LTS release: a Big Milestone for the Community
+
+The Apache Pulsar community announced the release of Apache Pulsar 3.0, the first Long-Term Support (LTS) version! Starting from Pulsar 3.0, the Pulsar community plans to release LTS versions to meet the needs of different users for stability and new features, as well as to reduce the burden of maintaining historical releases.
+
+The previous release process has short maintenance cycles of approximately 3 to 4 months, while many users are still using old versions. To keep up with new updates and features, they may be forced to perform upgrades within a short timeframe, for which they are not prepared in terms of available time and required efforts.
+
+Therefore, the Pulsar community introduces LTS versions with feature releases between them. The project follows a variant of Semantic Versioning, replacing `major.minor.patch` with `LTS.feature.patch`. For example:
+- 2.11.0 is a feature release
+- 3.0.0 is the first LTS release
+- 3.0.1 is a patch release of the LTS release
+- 3.1.0 is a feature release
+- 3.2.0 is a feature release
+- 3.2.1 is a patch release
+- 4.0.0 is an LTS release
+
+This pattern provides version support for users seeking stability and those seeking new features. Users who want a more stable release can use versions 3.0.x, while those seeking new features can use versions 3.x. This new release model is a major step for the Pulsar community because it:
+- Allows users to choose between different releases based on their needs for stability or new features;
+- Clarifies the release cycle for both maintainers and users;
+- Frees maintainers from spending too much time maintaining a long list of old releases.
+
+With the new release model, the Pulsar community looks to release LTS versions every 18 months, with bug fixes continuing for 24 months and security vulnerability patches supported for 36 months. See the image below for details.
+
+![](/img/pulsar-new-release-model.png)
+
+For more information, see [PIP-175](https://github.com/apache/pulsar/issues/15966) and the [Release policy](pathname:///contribute/release-policy/).
+
+## New Website
+
+The Apache Pulsar website has a fresh look! Thanks to Emidio Cardeira, Asaf Mesika, Tison Chen and Kiryl Valkovich for creating an engaging design that captures the future feel of our thriving community and next-gen solution.
+
+## Spring for Apache Pulsar 1.0.0 goes GA
```

Comment:
Thanks for adding this in @jak78 . Would you mind if we also point out that this is available in Spring Boot 3.2.0 something like this:

> ## Spring for Apache Pulsar
> [Spring for Apache Pulsar](https://docs.spring.io/spring-pulsar/docs/current/reference/reference/pulsar.html) provides a `PulsarTemplate` for publishing to a Pulsar topic, a `PulsarListener` annotation for consuming from a Pulsar topic, as well as various convenience APIs for Spring developers to ramp up their development journey into Apache Pulsar. Support is also included in Spring Boot via [auto-configuration](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#messaging.pulsar).
> 

Wdyt?

Reply:
Thanks! I just added your suggestion

Reply:
Thanks @jak78 ,
Looks good. 

Another subtle change I was suggesting was to change title from 
```
Spring for Apache Pulsar 1.0.0 goes GA
```
to
```
Spring for Apache Pulsar
```
as it removed any info about timeline and instead just mentions the framework. But no biggie either way, I am grateful that you mentioned the framework, regardless of any edits/tweaks. :)

Reply:
Done